src/main/java/nl/novi/techiteasydto/models/RequestTelevisionDto.java

```diff
@@ -0,0 +1,144 @@
+package nl.novi.techiteasydto.models;
+
+public class RequestTelevisionDto {
```

Comment:
Je hebt in deze input DTO nog geen validatie toegpast, zoals `@`NotNull of `@`Min(value=100)